.pre-commit-config.yaml

```diff
@@ -37,3 +37,12 @@ repos:
     - id: check-toml
     - id: check-yaml
     - id: detect-private-key
+
+  - repo: local
+    hooks:
+    - id: check-docs
+      name: check-docs
+      entry: scripts/check-docs.sh
```

Comment:
Could you call `just generate-docs` directly here?

Very cool to see local repo bits in use!

Reply:
No (at least, not just with `entry: just generate-docs`, which is what I tried)